frontend/src/components/ShootWorkers/ContainerRuntime.vue

```diff
@@ -7,16 +7,20 @@ SPDX-License-Identifier: Apache-2.0
 
 <template>
   <div class="d-flex flex-row">
-    <v-select
-      color="primary"
-      item-color="primary"
-      :items="criItems"
-      :error-messages="getErrorMessages('criName')"
-      @input="onInputCriName"
-      @blur="$v.criName.$touch()"
-      v-model="criName"
-      label="Container Runtime"
-    ></v-select>
+    <hint-colorizer hint-color="warning">
+      <v-select
+        color="primary"
+        item-color="primary"
+        :items="criItems"
+        :error-messages="getErrorMessages('criName')"
+        @input="onInputCriName"
+        @blur="$v.criName.$touch()"
+        v-model="criName"
```

Comment:
We should avoid this pattern 
```
@input="onInput"
v-model="value"
```
If we want to handle the input event we should create a getter and setter property `internalValue`. If we use @input we should use `:value` instead of v-model but register two input handlers in parallel. I know: "Not related to this PR but I want to mention it again".